docs/contributing/debug.md

```diff
@@ -40,7 +40,7 @@ outside the developer's machine and is not recommended.
       ```
    3. Run the daemon through the `make.sh` script:
       ```bash
-      $ ./hack/make.sh run
```

Comment:
Unrelated change snuck in; I don't think this even applies here as `run` as there's no 'run' bundle, run just relies on a PATH lookup.

Reply:
I wasn't sure if it was worth opening a PR for that little change, that's why I committed it separately from the fix but still bundled both in this PR.

Actually `KEEPBUNDLE=y` is needed because `./hack/make.sh` starts by deleting the `bundles/` dir and the `./hack/make/run` file doesn't have the execute bit, so it can't be run manually. Thus the only way to start a new delve session without rebuilding is `KEEPBUNDLE=y ./hack/make.sh`.

I'll open a new PR for that if you still think that's required :wink:

Reply:
Ah, yeah, this should probably be a separate PR. My usual invocation is `./hack/make.sh binary install-binary run`, but I think that we should probably discuss making this flow more ergonomic in general.